frontend/tests/controllers/QualityNominationTest.php

```diff
@@ -513,6 +513,72 @@ public function testDemotionCanBeResolvedAndLaterRevertedByReviewer(
         );
     }
 
+    public function testUpdateQualityNominationByReviewer() {
+        $problemData = \OmegaUp\Test\Factories\Problem::createProblem();
+
+        // Login as a reviewer and update QualityNomination.
+        $reviewerLogin = self::login(
+            \OmegaUp\Test\Factories\QualityNomination::$reviewers[0]
+        );
+
+        $getReviewedDataReviewer = \OmegaUp\DAO\QualityNominations::getReviewedData(
+            $problemData['problem']->problem_id,
+            \OmegaUp\Test\Factories\QualityNomination::$reviewers[0]->user_id
+        );
+        $this->assertEquals(
+            false,
+            $getReviewedDataReviewer['quality_seal'],
+        );
```

Comment:
Se podrá usar esta función:
```suggestion
        $this->assertFalse($getReviewedDataReviewer['quality_seal']);
```
?